addons/account/tests/test_account_inalterable_hash.py

```diff
@@ -188,3 +188,81 @@ def test_account_move_hash_versioning_v1_to_v2(self):
         moves_v1_bis.with_context(hash_version=1).action_post()
         integrity_check = moves.company_id._check_hash_integrity()['results'][0]
         self.assertEqual(integrity_check['msg_cover'], f'Corrupted data on journal entry with id {moves_v1_bis[0].id}.')
+
+    def test_account_move_hash_v2_floating_point(self):
```

Comment:
It's nice to have this test and see that it is green while merging, but it is not achieving anything for the CI later on

Reply:
The next test does that.
This test makes sure that it was failing before the fix, which we don't really care about. We just don't want it to fix in the future.